addons/binding/org.openhab.binding.network/OSGI-INF/NetworkHandlerFactory.xml

```diff
@@ -9,7 +9,10 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.network.internal.NetworkHandlerFactory">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
+    immediate="true"
+    configuration-policy="optional"
+    name="org.openhab.binding.network">
```

Comment:
I've seen this in other binding updates, that the component name of the handlerFactory is shortened to the bindings package name.